docsrc/installation.rst

```diff
@@ -35,12 +58,17 @@ To install the current develop branch from GitHub:
     pip install -e git+https://github.com/stan-dev/cmdstanpy@/develop
 
 
+If CmdStanPy is installed in this manner, 
+**you must install CmdStan**. The recommended way for Pip users 
+to do so is via the ``install_cmdstan`` function 
+:ref:`described below<install-cmdstan-fun>`
+
 *Note for PyStan users:*  PyStan and CmdStanPy should be installed in separate environments.
 If you already have PyStan installed, you should take care to install CmdStanPy in its own
 virtual environment.
```

Comment:
This is not true anymore (unless users use RTools to compile their models)

Reply:
Does the updated text in the latest commit work, or should we just remove this paragraph?